### ISSUE TITLE
tool does not restore the previous ON state neither for WiFi not for BT

### ISSUE THREAD

**Issue by cheshi-mantu** [stefansundin/bluesnooze#9]
M1, MB pro 16: Ventura 13.2.1

initial state: Wifi ON BT ON
settings: 
wifi **turn off**, BT **turn off**
wifi **restore initial state**, **BT restore initial state**
**actual result**: WIFI is **OFF** after wake up, BT is **OFF** after wake up

initial state: Wifi ON BT ON
settings: 
wifi **turn off, BT **turn off**
wifi **always enable**, BT **always enable**
**actual result**: WIFI is **ON** after wake up, BT is **ON** after wake up

initial state: Wifi ON BT ON
settings: 
wifi **do nothing**, BT **turn off**
wifi **do nothing, BT **restore the previous state**
**actual result**: WIFI is **ON** after wake up, BT is **OFF** after wake up

it looks like the restoration of the initial state on both – BT and WIFI does not work as expected well, at least in my case.
there is similar issue in the original repo: https://github.com/odlp/bluesnooze/issues/18

**Comment by stefansundin** [comment]
Hello. I haven't noticed anything broken recently but I'll try to test your scenarios at some point.

**Comment by BetterB0y** [comment]
Yes. I can confirm. Sometimes bluetooth is off after wake up. I have "restore previous state" set.

**Comment by vk0novalov** [comment]
Hi, I can confirm too on Air M1 with latest macOS - it's a random and rare bug. Maybe [this](https://github.com/odlp/bluesnooze/issues/18#issuecomment-1646600809) solution from origin project will be useful.

**Comment by warvariuc** [comment]
Also doesn't work for me on MacBook Pro M1
<img width="374" alt="image" src="https://github.com/user-attachments/assets/5afdb1de-03c5-4a71-a8eb-dacaced6b6d9">
